pkg/dataprotection/backup/request.go

```diff
@@ -193,7 +193,7 @@ func (r *Request) buildBackupDataAction(targetPod *corev1.Pod, name string) (act
 			Name: name,
 			ObjectMeta: metav1.ObjectMeta{
 				Namespace: r.Namespace,
-				Name:      r.Name,
+				Name:      GenerateBackupStatefulSetName(r.Backup, r.Target.Name, BackupDataJobNamePrefix),
```

Comment:
For compatibility reasons, use r. name, but when it exceeds 52, truncate the subsequent string